README.md

```diff
@@ -340,16 +481,51 @@ After writing to the captureOuput.pcap file you can read the file with the follo
 
 `tshark -r captureOutput.pcap`
 
-TShark offers a plethora of options to help specify any search vectors. A few examples of some options would be as follows.
+With having some basics covered, we may now get into some scripts written within different programming languages. Some languages that will be covered are BASH, Python, and Ruby.  
 
-- `-a| --autostop <capture autostop condition>` which tells TShark when to stop writing to the capture file. 
+### BASH
 
-- `-c <capture packet count>` which sets the maximum number of packets to read when capturing live data. 
+`function save_ping() {
+    tshark -w ping.pcap -f "host 8.8.8.8" -c 1
```

Comment:
spacing looked a little funky with the bash script

Reply:
https://www.freecodecamp.org/news/how-to-format-code-in-markdown/ <- helpful example